pecan/core.py

```diff
@@ -221,7 +221,7 @@ class PecanBase(object):
 
     def __init__(self, root, default_renderer='mako',
                  template_path='templates', hooks=lambda: [],
-                 custom_renderers={}, extra_template_vars={},
```

Comment:
These don't have defaults.

Also, `pecan.templating.RendererFactory` has the same problem in its constructor.